docs/getting-started/cli-reference.rst

```diff
@@ -0,0 +1,101 @@
+.. _cli-reference:
+
+CLI Reference
```

Comment:
Thanks for adding this to the docs site, it's definitely helpful to have on the site docs as well as on the CLI itself.